blocks/header/renderAuthDropdown.js

```diff
@@ -92,6 +92,7 @@ export function renderAuthDropdown(navTools) {
           width="25"
           height="25"
           viewBox="0 0 24 24"
+          aria-label="My Account"
```

Comment:
Is there an equivalent translation from the placeholders we can us instead?

Reply:
@sirugh I think that's a whole different task since the Boilerplate is not using placeholders on navigations.

Reply:
oh i didnt realize that. I thought you could use the useText tool. 🤷🏼

Reply:
Ok I see - it's not exposed so only available internally to dropins at this time. 👍🏼

Reply:
To be clear, you can use a placeholder in Blocks with no problem. That work just hasn't happened yet.